mk/docker.mk

```diff
@@ -34,7 +34,7 @@ REVISION 	 ?= $(shell git rev-parse HEAD)
 include $(CURDIR)/versions.mk
 
 # Supported OSs by architecture
-AMD64_TARGETS := ubuntu20.04 ubuntu18.04 ubuntu16.04 debian10 debian9
+AMD64_TARGETS := ubuntu22.04 ubuntu20.04 ubuntu18.04 ubuntu16.04 debian10 debian9
```

Comment:
Same comment as on https://github.com/NVIDIA/nvidia-container-toolkit/pull/1004

Do we want to add arm64 builds here too?